packaging/installer/methods/freebsd.md

```diff
@@ -7,8 +7,17 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
 
 # Install Netdata on FreeBSD
 
-You can install Netdata from either the `ports` or `packages` collections.
+## Install from ports/packages
+You can install Netdata from either the `ports` or `packages` collections. To install from packages:
+```sh
+# pkg install netdata
+```
+You also need to enable the netdata service in `/etc/rc.conf` (add `netdata_enable="YES"`) and start the service:
+```sh
+# service netdata start
+```
 
+## Install latest version
```

Comment:
I think it's more correct to differentiate between ports/packages and source.

```suggestion
## Install from source

```

Reply:
Actually installing from 'ports' also is installing from source. So probably 'latest source version' may be correct.

Reply:
See: https://www.freshports.org/net-mgmt/netdata/
Installing from ports should work like this: `cd /usr/ports/net-mgmt/netdata/ && make install clean`
But I did not test and I'm not an experienced FreeBSD user.

Reply:
I will give additional information to complement this conversation, before to write this comment, I ran the following command:

```
portsnap fetch extract
```

When FreeBSD finished the extraction process, I verified that right now we have `Netdata 1.20` inside `/usr/ports/net-mgmt/netdata`, while we released Netdata 1.21 last Monday.

Reply:
> See: https://www.freshports.org/net-mgmt/netdata/
> Installing from ports should work like this: `cd /usr/ports/net-mgmt/netdata/ && make install clean`
> But I did not test and I'm not an experienced FreeBSD user.

These are the steps to compile, you are right.

Reply:
This is probably because we currently don't maintain the FreeBSD ports tree or packages. We _shoudl_ and we'll get there.